### PR TITLE
Landing pages: a11y enhancements 

### PR DESCRIPTION
**Pull request by jeffgolenski**
Fixes: https://github.com/Automattic/jetpack/issues/4467

a11y enhancements for the apps landing page. adding title tags, alt tags, aria labels.

enhancements for the plans landing pages will come in another PR. 

No visual changes.

### CODE REVIEW

File: _inc/client/apps/index.jsx [hunk 1]
```diff
@@ -147,7 +147,7 @@ const Apps = ( props ) => {
 						{ __( 'Get WordPress apps for any screen.' ) }
 					</p>
 
-					<img src={ imagePath + '/apps/triple-devices.svg' } className="jp-landing-apps__devices" />
+					<img src={ imagePath + '/apps/triple-devices.svg' } className="jp-landing-apps__devices" role="img" alt="Example of three devices to use the WordPress apps. An iPhone, Android phone, and a apple laptop computer." />
```
Comment (eliorivero) [inline _inc/client/apps/index.jsx]: Make alt attribute available for translation like `alt={ __( 'string to translate' ) }`

File: _inc/client/apps/index.jsx [hunk 2]
```diff
@@ -156,7 +156,7 @@ const Apps = ( props ) => {
 
 					<p className="jp-landing-apps__btn-container">
 						<Button href="http://itunes.apple.com/us/app/wordpress/id335703880?mt=8" title={ __( 'WordPress.com in the App Store' ) } className="jp-app-button button-ios">
-							<svg width="28" height="28" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 96 96"><path d="M74.7,62.2c-3.5-3.2-5.2-7.2-5.3-12.1c-0.1-6.2,2.8-11.1,8.5-14.5c-3.2-4.5-8-7.1-14.4-7.6c-2.4-0.2-5.2,0.3-8.7,1.5
+							<svg width="28" height="28" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 96 96" aria-labelledby="wpcomIosbtn" role="img"><title id="wpcomIosbtn">Download the free WordPress app for your iPhone</title><path d="M74.7,62.2c-3.5-3.2-5.2-7.2-5.3-12.1c-0.1-6.2,2.8-11.1,8.5-14.5c-3.2-4.5-8-7.1-14.4-7.6c-2.4-0.2-5.2,0.3-8.7,1.5
```
Comment (eliorivero) [inline _inc/client/apps/index.jsx]: This and other title tags should be available for translation.

**Comment by jeffgolenski** [comment]
@eliorivero Good eye. I've added all the translation syntax to the title and alt tags. Can you give it another look?

**Comment by dereksmart** [comment]
merged to 4.5 479bf71

**Pull request merged**